projects/scapy/project.yaml

```diff
@@ -5,6 +5,7 @@ primary_contact: "guedou@gmail.com"
 auto_ccs:
   - "jvoisin@google.com"
   - "ipudney@google.com"
+  - "evverx@gmail.com"
```

Comment:
Just confirming adding @evverx is okay @guedou -- @evverx did you get the confirmation in private?

Reply:
> did you get the confirmation in private?

@DavidKorczynski no I didn't. I tagged the maintainers to discuss it here.

Reply:
(a scapy maintainer here)
Why do you need that @evverx, if I may ask?

Taking a step back, I don't understand what this fuzzing test is trying to achieve. (I must say I'm not even sure what the goal is here. Is it to fuzz scapy entirely? or is google using our pcap parser and therefore has interest in fuzzing it?)

Fuzzing `rdpcap()` is far from being the best entry point if you want to fuzz scapy's core or layers (which is most of the code). Actually what are you looking for when fuzzing? Crashes? Hangs? Returns ? Crashes in dissectors are silent (converted to raw), because we expect crappy packets on the network. So feeding garbage into Scapy will generally not do much. The best things you could find with fuzzing are probably infinite loops, which does interest us a lot, but again I would rather just fuzz `scapy.layers.l2.Ether(<garbage>)` that `rdpcap`.

Now, regarding the issues that this project reports (the only ones we've got so far are exclusively related to our pcap parser, which is kinda expected), I would rather have guedou receive them and pass it to us maintainers (usually opens an issue, but who knows if it's critical), so I don't really get why you'd want that.

Reply:
> Why do you need that @evverx, if I may ask?

Usually if I change something I kind of keep maintaining it and it's hard to maintain fuzzers when their results are hidden. In reality since anyone can run those fuzz targets it doesn't matter much whether issues are hidden or not (it was kind of discussed in https://github.com/google/oss-fuzz/issues/8921#issuecomment-1356785464 for example).

> Actually what are you looking for when fuzzing?

I think I spent a couple of days recently looking at `pack/unpacks` and thought that this fuzz target would be the best place to make sure that say `raw(p.__class__(raw(p))) == raw(p)` so to me it's not about crashes but about making sure that some assumptions are more or less correct.

> I don't understand what this fuzzing test is trying to achieve.

As far as I can see it was discussed in https://github.com/secdev/scapy/issues/3039 and looking at the fuzz target I would assume it was just a simple fuzz target to get the ball rolling. It doesn't actually fuzz anything and hasn't changed since then though.